kolibri_instant_schools_plugin/assets/src/views/SideNav/UserProfileSideNavEntry.vue

```diff
@@ -27,7 +25,7 @@
       CoreMenuOption,
     },
     $trs: {
-      profile: 'Profile',
+      account: 'Account',
```

Comment:
I hadn't considered the fact that we might need to have this plugin translated. If we can do this on Crowdin then it's good that we're thinking of this now, but most or all users won't use the platform in English.

We'd need Swahili, Kishwahili (aka Swahili Tanzania), French and possibly Portuguese. 

cc @indirectlylit @jeepurs

Reply:
Now that I've gone further along in code review, I see that @ralphiee22 has actually edited the translation files! Didn't know Ralphie spoke those languages :-p